modules/foreman_search_facts.py

```diff
@@ -70,7 +70,7 @@
     search: name = http_proxy
   register: result
 - debug:
-    var: result.response[0].value
+    var: result.resources[0].value
```

Comment:
Has it always been resources? Or did this change with a foreman version?

Reply:
it was always resources, as it's in the module code ;)

Reply:
Lucky us. Someone is looking at this stuff from time to time. ;)

Reply:
I can neither confirm nor deny that I was staring at my `apypie` branch for 5 minutes, wondering why it did not return me any results ;)